torch/csrc/jit/passes/quantization.cpp

```diff
@@ -364,9 +365,13 @@ void InsertObserversHelper::insertObservers(
 
   script::Method method = module.get_method(method_name);
   auto graph = method.graph();
+  // To cleanup traced graph
+  ConstantPooling(graph);
   ConstantPropagation(graph);
   // must do constant propagation first before replacement
   replaceConvolutionWithConv2d(graph);
+  // fuse decomposed linear into aten::linear
+  FuseLinear(graph);
```

Comment:
Though for this pass I remember there was some back-and-forth earlier about how it's not profitable in some cases, and might mess up some downstream systems (e.g. CUDA fuser). We could add it generally and see if anything breaks

Reply:
yeah we can't do this by default since it doesn't work for autodiff, but @wanchaol is working on this.